R/utils.R

```diff
@@ -1,3 +1,33 @@
+is_integer <- function(x) {
+  is.integer(x) && assertthat::is.scalar(x)
+}
```

Comment:
If you are not aware, there is also predicate in `rlang`

```r
rlang::is_integer(x, n = 1L)
```
should be equivalent of what is above here

Reply:
I was not aware of `rlang::is_integer()` (too much verbs in tidyverse for me :smile:). Thanks!

Reply:
done in https://github.com/RLesur/crrri/pull/40/commits/336afef7d645d2e6083ea7e8bcb0dd03efc4038c